infinispan-remote/src/main/java/org/hibernate/ogm/datastore/infinispanremote/impl/protobuf/ByteProtofieldAccessor.java

```diff
@@ -11,29 +11,24 @@
 
 /**
  * There is no explicit "byte" support in Protostream.
- * Decided to encode it as a byte array type; on reading a longer sequence it will be truncated as we only read the first byte.
+ * Decided to encode it as a int type; on reading an int will be truncated as a byte,
+ * according to the endian behavior adopted by the {@link Integer#byteValue()} method
```

Comment:
What _endian behaviour_ ?

We need to make sure the encoding is the same independently from the specific platform this is running on.

I assumed the byteValue() method provided a stable specification, but either I'm wrong or this comment is overly concerning :)

Reply:
this comment is definitely overly concerning :) I think we can rely on a consistency of Integer#byteValue(). I will change it. Thank you :)